Gems/Multiplayer/Code/Source/Pipeline/NetworkPrefabProcessor.cpp

```diff
@@ -46,7 +45,7 @@ namespace Multiplayer
     {
         if (auto* serializeContext = azrtti_cast<AZ::SerializeContext*>(context); serializeContext != nullptr)
         {
-            serializeContext->Class<NetworkPrefabProcessor, PrefabProcessor>()->Version(1);
+            serializeContext->Class<NetworkPrefabProcessor, PrefabProcessor>()->Version(2);
```

Comment:
To me magic numbers should have some context / hints at how set. '2' in this context is just confusing, consider moving to at least a named const_expr

Reply:
The version update here is simply to enforce all existing spawnables to rebuild by the Asset Processor. There's no actual meaning for 2 here.

Out of curiosity I have searched through the code base and there are 0 occurrences where Version() has anything other than [0-9]. 
There are 2378 cases where a hardcoded number is passed into Version().